meta-mender-c444/templates/local.conf.append

```diff
@@ -13,9 +13,13 @@ MENDER_DEVICE_TYPES_COMPATIBLE = "imx8mq-itx-p-c444"
 PREFERRED_PROVIDER_u-boot = "u-boot-imx"
 IMAGE_CLASSES += "mender-setup-c444"
 
+# SD and eMMC boot are mutually exclusive. Please select only one, either SD boot or eMMC boot
+
+# Boot from eMMC
 # Uncomment the following 2 lines to boot from eMMC on imx8mq-itx-p-c444
```

Comment:
eMMC is already in effect (_i.e., uncommented_). Hence the following comment/line shall be removed 

```suggestion
```

—-

CC @tkmozhi